src/include/function/gds/gds_task.h

```diff
@@ -13,13 +13,16 @@ struct FrontierTaskInfo {
     graph::Graph* graph;
     common::ExtendDirection direction;
     EdgeCompute& edgeCompute;
+    std::optional<common::idx_t> edgePropertyIndex;
```

Comment:
I would try to put this in somewhere else. Doesn't sound like a field belong to FrontierTaskInfo

Reply:
I think this is suggested by @benjaminwinger ,you may want to talk to him?